dali/server/daldap.cpp

```diff
@@ -131,7 +131,8 @@ class CDaliLdapConnection: implements IDaliLdapConnection, public CInterface
         {
             username.append(filesdefaultuser);
```

Comment:
We suspected it (filedefaultuser) could be blank (since it's not configured in any cloud system), and taking this commit into my test image that proved to be the case.

As discussed offline, I don't think there's any sensible case for allowing this to continue if filesDefaultUser is empty here (which we've seen it do).
If empty, it should return SecAccess_None

However, I am worried what it might break. Nothing *should* be relying on this code path, but...

Discussed briefly with Gavin earlier and we think there should be an option added to in effect disable the defaultuser functionality altogether (with the option defaulted off for now).
Then it be turned on via config and tested (in e.g. our TFE clusters), to discover what is relying on the default user.

I don't think there's much value in merging this one right now, we know it's always blank in cloud.
@kenrowland - can you open a new JIRA to add an option to conditionally disable use of the default user via configuration? Thanks.